### PR TITLE
DT-1662 Show agency in summary (& nearby)

### PR DESCRIPTION
**Pull request by Macroz**
Metro 'M' should be refactored sometime.

### CODE REVIEW

File: app/component/summary-row.scss [hunk 1]
```diff
@@ -90,8 +90,9 @@
     flex-grow: 1; //Needed on IE
 
     .leg {
-      flex: 1 0 0;
```
Comment (hannesj) [inline app/component/summary-row.scss]: Or use https://github.com/luisrudge/postcss-flexbugs-fixes

Comment (Macroz) [inline app/component/summary-row.scss]: Seemed to me that the page worked fine even without the fix so I have removed it for now.

File: app/component/RouteNumberContainer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  } else if (showAgency && route.agency) {
+    return route.agency.name;
+  }
+  return '';
+};
+
+const getMode = route =>
+  route.mode || (route.rentedBike && 'CITYBIKE');
+
+const SmartRouteNumber = ({ route, ...props }) =>
```
Comment (hannesj) [inline app/component/RouteNumberContainer.js]: Should be RouteNumberContainer?

**Comment by hannesj** [comment]
The issue catched by tests seems to be a real issue

**Pull request merged**